src/tests/ftest/util/command_utils.py

```diff
@@ -175,9 +175,10 @@ def stop(self):
 
         """
         if self._process is not None:
-            # Send a SIGTERM to the stop the subprocess and if it is still
-            # running after 5 seconds send a SIGKILL and report an error
-            signal_list = [signal.SIGTERM, signal.SIGKILL]
+            # Send a SIGTERM to stop the subprocess and if it is still
+            # running after 5 seconds give it another try. If that doesn't
+            # stop the process send a SIGKILL and report an error
+            signal_list = [signal.SIGTERM, signal.SIGTERM, signal.SIGKILL]
```

Comment:
Is this s typo? We have `signal.SIGTERM` in the list?

Reply:
No, it is not a typo. What I observed is that the SIGTERM signal is not being propagated through the test client node so the IOR command is never killed gracefully in that node. But when sending 2 SIGTERM signals both processes are killed and the test pass.

Reply:
I think this was not the case before. Any idea why do we need 2 sigterm signals?

Reply:
No, that was not the case before. It is strange but could not figure it out. I saw the SIGTERM signal was not propagating to the test client node.
I tried to capture the signal in the command_utils.py to see if it was at least reaching the process but could not.
Any idea from your side?

Reply:
Nope I don't. I remember when I first submitted the test it used to work with single SIGTERM signal.

Reply:
Opened the ticket DAOS-6850 reporting this issue and based on the triage meeting, this is a known issue with MPI applications and not directly related to DAOS. It may require more than one ^C to get the termination propagated to all of the ranks. Not something we can fix/address within DAOS.